llvm/include/llvm/IR/DerivedTypes.h

```diff
@@ -685,13 +685,19 @@ class PointerType : public Type {
 
   /// This constructs a pointer to an object of the specified type in a numbered
   /// address space.
+  LLVM_DEPRECATED("PointerType::get with pointee type is pending removal. Use "
+                  "Context overload.",
+                  "PointerType::get(LLVMContext, AS)")
```

Comment:
Please use `[[deprecated]]` instead. Only use LLVM_DEPRECATED if you can actually provide a working fixit, which you can't do here.

Reply:
Thanks for the heads up! Taken care of in b3c03d6bdad9f82834984f6aff0bca9c19af8afb